frontend/app/src/components/status/notifications/NotificationPopup.vue

```diff
@@ -58,22 +58,25 @@ const { dark } = useTheme();
       @dismiss="dismiss(visibleNotification.id)"
     />
     <template v-if="queue.length > 0">
-      <div class="flex justify-end border-t border-default">
-        <RuiTooltip open-delay="400">
+      <div
+        class="flex justify-between p-2 items-center border-t border-default"
+      >
+        <RuiTooltip open-delay="400" :popper="{ placement: 'right' }">
           <template #activator>
-            <RuiButton variant="text" class="!p-2" icon @click="dismissAll()">
-              <RuiIcon name="menu-unfold-line" />
-            </RuiButton>
+            <RuiChip class="!p-1.5" color="primary" size="sm">
+              {{ queue.length }}
+            </RuiChip>
           </template>
-          {{ t('notification_popup.dismiss_all') }}
+          <span v-text="t('notification_popup.tooltip')" />
```

Comment:
@lukicenturi, @tewshi in general if you find these `v-text` html elements while working replace them, and also we should also get rid of the useless span/div with only text in tooltips